requirements.txt

```diff
@@ -1,7 +1,23 @@
 asgiref==3.2.3
-Django==3.0.3
 pytz==2019.3
 sqlparse==0.3.0
+certifi==2019.11.28
+chardet==3.0.4
+Click==7.0
 commonmark==0.9.1
+dj-database-url==0.5.0
+Django==2.1.5
 django-heroku==0.3.1
+Flask==1.1.1
 gunicorn==20.0.4
+idna==2.8
+itsdangerous==1.1.0
+Jinja2==2.11.1
+MarkupSafe==1.1.1
+Pillow==7.0.0
+psycopg2==2.8.4
+pytz==2019.3
+requests==2.22.0
+urllib3==1.25.7
+Werkzeug==1.0.0
+whitenoise==5.0.1
```

Comment:
Please revert all the changes here except for adding Pillow, this looks like the contents of your personal pip.